gson-problem/pom.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>problem-parent</artifactId>
+        <groupId>org.zalando</groupId>
+        <version>0.24.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>gson-problem</artifactId>
```

Comment:
There was no module right now that you could've used as a blueprint, but my preferred naming convention for modules/artifacts would be `problem-{name}`.

The jackson integration deviates from that because jackson has a common naming for external data types/modules.

Reply:
Sure, I'll update it to `problem-gson`.